docs/API_README.md

```diff
@@ -0,0 +1,142 @@
+## API Docs
+
+**Interface to sign in**
+
+| URI      | Method | Authorization |
+|----------|--------|---------------|
+|`/token/` | `POST` | not required  |
+
+
+Params:
+ * `email=[string]`
+ * `password=[string]`
+
+Success Response:
+
+* Code: 200
+* Content: `{"refresh": "<refresh token>", "access": "<access token>"}`, where `<refresh token>` and `<access token>` are the valid JWT tokens.
+
+Specific Error Response
+* Code: 400
+* Content: `{"detail": "No active account found with the given credentials"}`
+* How to solve: register the user or provide the valid credentials.
+
+Sample Call:
+
+`curl -X POST http://127.0.0.1:8888/token/ -d "email=some-email&password=secret"`
+ 
+-----------
+   
+**Interface for refreshing an access token**
```

Comment:
Make this and other similar headlines like
```
#### Interface for refreshing an access token
```

It's necessary to be able to reference to them. Otherwise, it's not possible.